docs/stacks-guide-gcp.md

```diff
@@ -483,7 +483,7 @@ where we left off.
 
 [gcp]: https://cloud.google.com/
 
-[stacks-guide-continue]: stacks-guide.html#install-support-for-our-application-into-crossplane
```

Comment:
It looks like this existing format works when the docs are rendered, but I like the change for consistency!